docs/api/search.md

```diff
@@ -0,0 +1,107 @@
+Search
+======
+
+####Our api supports searching artifact metadata which will return a link header for each search result.
+
+####Note:
+    * The only requirement is the search portion of the criteria.
```

Comment:
This star stuff is not displaying correctly.  I think it is because you have it indented.  So it thinks its in a code block.  Can you outdent all this?